### PR TITLE
feat(session-flow): a "You are here" position panel on the save-point skills

### PR DESCRIPTION
**Pull request by kyle-sexton**
No linked issue

## Summary

Both save-point skills showed the operator exactly two things: a ticked enforcement checklist, which
is the skill's own audit trail, and the rails resume prompt, which is a block to copy. Everything
answering *what did we do, where are we, what is next* was computed and then filed into the handoff
document — whose stated reader is "a session with NO prior context" (`reference/structure.md`), so
the operator never reads it. On the prompt-only path no file is written at all and the recap existed
nowhere.

At the moment a human is deciding whether this is a sane place to stop and whether the work is still
pointed where they wanted it, the skill showed them a compliance checklist. The information to answer
both was already gathered — it was just never rendered for them.

## Fix

A new engine section, `Emit the position panel`, owns it once for both citing skills:

```text
**You are here**

  [x] Phase 1 — discovery
  [x] Phase 2 — engine
▸ [~] Phase 3 — wiring          you are here
  [ ] Phase 4 — evals
  [ ] Phase 5 — docs

3 of 5 phases complete · completion criteria 4/7 met (2 UNVERIFIED)

Done this session — retry wrapper landed and green (a1b2c3d); OrderWriter stub does not compile yet.
Where we are — mid Phase 3, blocked on that stub.
Up next — finish the cancellation pass-through, then Phase 3 edge-case tests (§11 owns the rest).
```

**It restates; it never discovers.** Every line comes from what `Locate the position first` and the
sections above already established. It triggers no read the save-point did not already need — that is
the line between it and `orient`, which sweeps durable and off-thread state on demand.

**Units are resolved from the work, not assumed.** A first-match ladder takes workflow-checklist
stages, then plan/spec/PRD phases, then an issue chain, then live `TaskList` items, then completion
criteria. Work matching none of those gets the three prose blocks and *explicitly no rail* —
inventing phases to have something to draw renders a map of a plan that does not exist, and the
operator would resume against it.

**The rail is vertical because a horizontal one wraps.** One unit per line, one line per block, never
a continuation line. A `→`-chained row wraps at whatever width the terminal happens to be, and the
wrap orphans the position marker from the unit it marks — destroying the single thing the panel
exists to show. Above 8 units the middle elides to a `… N more` line, keeping the ends and the
current position; the whole panel caps at 16 lines. Status glyphs are the ones `structure.md` already
uses for the TaskList snapshot, so no legend is needed.

**It cannot become a reason to lose the rails prompt.** The one observed failure of this engine is a
turn that ends before the prompt reaches the screen (`skills/handoff/context/gotchas.md`), and this
change puts new text in front of that prompt. So the caps are load-bearing rather than cosmetic, an
uncertain panel degrades to one abbreviated line instead of growing, and the engine states outright
that the panel never gates the rails. A gotchas entry names this as the same failure with a new
cause.

Delivery order per skill: `handoff` is panel → checklist → rails, leaving the rails-last rule intact;
`continue-in-background` is panel → rails → launch, and the launched agent still receives exactly the
text between the rails, never a line of the panel.

Not a detection-contract change — the panel sits above every keyed signal and outside the copy
region, so `find-handoff` recovers exactly what it recovered before and needs no edit. That is stated
explicitly in the contract section, which otherwise treats a shape change as a knowing break.

Claim provenance and the redaction sweep both extend to the panel: an inherited status keeps its
`UNVERIFIED (<source>)` marker, the completeness line says how many marks are unverified, and the
panel joins the outbound set the mandatory redaction pass covers.

## Verification

Run against the rebased tree, not an earlier one:

- `npx markdownlint-cli2 "plugins/session-flow/**/*.md"` — 0 issues
- `editorconfig-checker plugins/session-flow` — clean
- `typos plugins/session-flow` — clean
- `jq empty` on both changed eval suites and the manifest — parse clean
- `scripts/validate-plugins.sh` — all manifests and the catalog validated
- `scripts/check-changelog-parity.sh` in all four modes — `--check`, `--check-bump origin/main`,
  `--check-order`, `--check-preserved origin/main` — all pass
- `scripts/check-changed-skills.sh origin/main` — both changed skills PASS, 0 errors
- `node scripts/generate-catalog.mjs` and `generate-cheatsheet.mjs` — no drift; frontmatter
  `description` and `metadata.summary` were deliberately left alone, since this is an output-shape
  change rather than a new capability and both feed the skill-listing context budget
- session-flow's own three test suites (`check-usage-limit-reset`, `parse-transcript`, `observer`) —
  pass

New eval coverage grades the *shape* rather than mere presence: the vertical rail with the current
unit marked, elision above 8 units, the prose fallback inventing no units, and the launched agent's
payload staying panel-free.

Not verified here: the behavioral end-to-end (invoking the skills and reading the rendered output at
an 80-column width). The evals encode those assertions; a live run is a separate exercise.

Also unverified by me, and worth a reviewer's eye: `scripts/run-plugin-tests.sh` fails in
`plugins/claude-config/skills/audit-permission-state/scripts/` in my local container. That area is
untouched by this diff and `main` is green at the commit this branch was cut from, so I read those as
container-local (uid 0, and several of those suites read the ambient `~/.claude` state). CI is the
arbiter, not that reading.

## Related

Refs #3018 — that issue asks to compress the handoff instruction walk and names `save-point.md`
specifically. This change moves that file the other way, 452 → 547 lines (+95). Shipping the panel at
full spec was a deliberate call: the rationale paragraphs are what make the caps stick, and the
gotchas entry documents a real failure they guard against, so trimming them here is how the rule
quietly stops being followed. Recorded plainly so the compression pass starts from an accurate
picture rather than a surprise.

---
_Generated by [Claude Code](https://claude.ai/code/session_014cvjkNWz2hCfo6S5UBpgcv)_

### CODE REVIEW

File: plugins/session-flow/reference/save-point.md [hunk 1]
```diff
@@ -185,6 +185,95 @@ When the target file already exists on disk (extending an earlier turn's write),
...
+```text
+**You are here**
+
+  [x] Phase 1 — discovery
+  [x] Phase 2 — engine
+▸ [~] Phase 3 — wiring          you are here
+  [ ] Phase 4 — evals
+  [ ] Phase 5 — docs
+
+3 of 5 phases complete · completion criteria 4/7 met (2 UNVERIFIED)
```
[the remark this answers is not in the document]
Comment (kyle-sexton) [reply plugins/session-flow/reference/save-point.md]: Correct, and worth fixing twice over. Fixed in 3cac264.

The example now reads `2 of 5`, and the miscount is closed off with a rule rather than just a corrected number — because the example is the thing agents copy, and a silently-right example is one edit away from being wrong again:

> **The count is of COMPLETED units, and an in-progress unit is not one.** […] `[~]`, `[ ]`, and `[!]` all count against the total, and only `[x]` counts toward it. Rounding the current unit up is the one arithmetic a progress read is most tempted into and least allowed — it reports work as landed while the operator is looking at the line that says it is not.

Particularly bad in this section, whose entire purpose is giving the operator an accurate completeness read before they decide whether to stop.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: plugins/session-flow/reference/save-point.md [hunk 2]
```diff
@@ -185,6 +185,95 @@ When the target file already exists on disk (extending an earlier turn's write),
...
+different tasks. Take the FIRST that applies, and name the unit kind in the rail so the operator
+knows what they are looking at:
+
+1. Workflow-checklist stages — the stage ledger at `<memory_dir>/<slug>/` the sibling `workflow`
+   skill maintains.
+2. Phases named by a backing plan, spec, or PRD — the artifact "Locate the position first" already
+   read this turn.
+3. An issue chain — the parent work-item and its sub-issues.
+4. Live `TaskList` items — already fetched for [`structure.md`](structure.md)'s
+   `Environment to re-establish`, so this costs nothing extra.
```
[the remark this answers is not in the document]
Comment (kyle-sexton) [reply plugins/session-flow/reference/save-point.md]: Right on both halves — the rung asked for a list nobody fetched, while the rule above forbade fetching it. Unfollowable either way on that path. Fixed in 3cac264.

Rung 4 is now full-path only and falls through to completion criteria on prompt-only. That fall-through costs almost nothing, which is the part your comment surfaced indirectly: *"No non-trivial task list to reconstitute"* is already one of the criteria in "Choosing the path" that selects prompt-only. A session with a task list worth drawing was supposed to be on the full path, so the rung was never going to fire there in the auto-detected case.

The forced case is the real gap — `/handoff prompt` bypasses auto-detect, so that criterion was never tested and there may well be a live list. There the skill makes the one `TaskList` call, named explicitly as the single exemption to "never discovers":

> **One call is exempt, and only one:** the `TaskList` fetch a FORCED prompt-only save-point never made […] It reads the session's own task ledger rather than the world outside the conversation, so it cannot turn the panel into an orientation sweep — which is the thing this rule exists to prevent.

Preferred over guessing the list from the conversation, which is the failure the live-`TaskList` rule exists to prevent elsewhere in this engine. `handoff`'s prompt-only checklist tick was updated to match so the two surfaces cannot drift.

---
_Generated by [Claude Code](https://claude.ai/code)_
